app/src/components/common/Address.vue

```diff
@@ -21,34 +20,41 @@ import { formatAddress } from "src/filters"
 export default {
   name: `lunie-address`,
   filters: {
-    formatAddress,
+    formatAddress
   },
   props: {
     address: {
       type: String,
-      required: true,
+      required: true
     },
+    tooltipText: {
+      type: String,
+      required: true
+    }
   },
   data: () => ({
-    copySuccess: false,
+    copySuccess: false
   }),
   methods: {
     onCopy() {
       this.copySuccess = true
       setTimeout(() => {
         this.copySuccess = false
       }, 2500)
-    },
-  },
```

Comment:
Nooo, why is this changing again?

Reply:
probably dependencies not updated?

Reply:
ah yeah, that must be it. Older version of prettier. Ping @jbibla